docs/essentials/exceptions/exceptions-use-finally-blocks.md

```diff
@@ -1,8 +1,8 @@
-# How to use Finally blocks
+# How to use finally blocks
 
-When an exception occurs, execution stops and control is given to the closest exception handler. This often means that lines of code you expect to always be called are not executed. Some resource cleanup, such as closing a file, must always be executed even if an exception is thrown. To accomplish this, you can use a Finally block. A Finally block is always executed, regardless of whether an exception is thrown.
+When an exception occurs, execution stops and control is given to the closest exception handler. This often means that lines of code you expect to always be called are not executed. Some resource cleanup, such as closing a file, needs to be done even if an exception is thrown. To do this, you can use a Finally block. A Finally block always executes, regardless of whether an exception is thrown.
```

Comment:
> This often means that lines of code you expect to always be called are not executed

"lines of code" are not called, but executed.

You shouldn't expect code to be always executed, particularly if you call into other methods . This seems like a strange thing to write in the context of a platform that has exceptions.